api/HTMLMediaElement.json

```diff
@@ -131,10 +131,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
There's no "Chromium to 1" buddy PR for this, so I notice that the Chrome data for `addTextTrack` is wrong. It was in Opera 15 and Chrome 28, so this bit is correct. I trust there will be more PRs to fix the incorrect Chrome data.

As an aside, I don't believe there was ever a prefixed variant of this, so the data is wrong in multiple ways. There was an alternative name though, see https://bugs.webkit.org/show_bug.cgi?id=77381.